kaggle_environments/envs/lux_ai_2021/lux_ai_2021.py

```diff
@@ -136,19 +160,38 @@ def interpreter(state, env):
 
     ### 3.4 Handle finished match status
     if match_status["status"] == "finished":
-        player1.status = "DONE"
-        player2.status = "DONE"
+        if player1.status == "ACTIVE":
```

Comment:
another major bug. Previously, if one agent errors out and another doesn't match keeps running and both agents marked as done